lib/helper/WebDriver.js

```diff
@@ -906,15 +906,24 @@ class WebDriver extends Helper {
   }
 
   /**
-   * {{> grabTextFrom }}
+   * {{> grabTextFromAll }}
    *
    */
-  async grabTextFrom(locator) {
+  async grabTextFromAll(locator) {
     const res = await this._locate(locator, true);
     assertElementExists(res, locator);
     const val = await forEachAsync(res, el => this.browser.getElementText(getElementId(el)));
     this.debugSection('Grab', String(val));
-    return val.length > 1 ? val : val[0];
+    return val;
+  }
+
+  /**
+   * {{> grabTextFrom }}
+   *
+   */
+  async grabTextFrom(locator) {
+    const val = await this.grabTextFromAll(locator);
+    return val[0];
```

Comment:
What happens if returned empty array?

Reply:
If the array is empty it will return 'undefined' but if the value itself is 'undefined' it will surely throw an error. Will handle it, Thanks for pointing it out :)

Reply:
@Vorobeyko good point! 
At this moment everything will be fine, because `grabTextFromAll` has `assertElementExists(res, locator);` call to ensure that at least one element was found. However, it should not be part of `grabTextFromAll` method and should be moved to `grabTextFrom`

Reply:
@DavertMik I am internally calling grabTextFromAll(), So the assertion will happen any how. If we move this line assertion will not happen in grabTextFromAll() and it may create issues. Please Correct me If I understood wrongly.

Reply:
@Vorobeyko Your inputs?

Reply:
> . If we move this line assertion will not happen in grabTextFromAll() and it may create issues. Please Correct me If I understood wrongly.

`grabTextFromAll` may return an empty array, that's fine. At least you can have a guess that an array can be empty. 

But if `grabTextFrom` will return null instead of string this changes the behavior, so it is the problem we want to fix with this PR. That's why to make types consistent we need `assertElementExists` only in `grab*` methods.

Reply:
Made this changes